includes/Story_Renderer/HTML.php

```diff
@@ -352,10 +352,36 @@ protected function add_publisher_logo() {
 			return;
 		}
 
-		$publisher_logo = $story_element->getAttribute( 'publisher-logo-src' );
+		$publisher_logo_src = $story_element->getAttribute( 'publisher-logo-src' );
+		if ( empty( $publisher_logo_src ) || $publisher_logo_src === $this->get_publisher_logo_placeholder() ) {
```

Comment:
Ideally we avoid checking for the placeholder image here, but it seems like we don't store the data in a way that this is possible :-(